website/content/docs/commands/job/restart.mdx

```diff
@@ -0,0 +1,206 @@
+---
+layout: docs
+page_title: 'Commands: job restart'
+description: |
+  The job restart command is used to restart allocations for a job.
+---
+
+# Command: job restart
+
+The `job restart` command is used to restart or reschedule allocations for a
+particular job.
```

Comment:
Added some description about their difference closer to the top.

```
Restarting the job calls the [Restart Allocation][api_alloc_restart] API
endpoint to restart the tasks inside allocations, so the allocations themselves
are not modified but rather restarted in-place.

Rescheduling the job uses the [Stop Allocation][api_alloc_stop] API endpoint to
stop the allocations and trigger the Nomad scheduler to compute new placements.
This may cause the new allocations to be scheduled in different clients from
the originals.
```